lib/lotus/repository.rb

```diff
@@ -296,10 +249,14 @@ def persist(entity)
       #   created_article = ArticleRepository.create(existing_article) # => no-op
       #   created_article # => nil
       #
+      #   imported_article = ArticleRepository.create(Article.new(title: 'New Lotus features', id: 123))
+      #   imported_article.id  # => 123
       def create(entity)
-        unless _persisted?(entity)
+        begin
           _touch(entity)
           @adapter.create(collection, entity)
+        rescue Sequel::UniqueConstraintViolation, Lotus::Model::Adapters::Memory::Collection::UniqueConstraintViolation => e
```

Comment:
Totally agree here, we are letting the operation to silently fail. This shouldn't be caught.